src/resources/invoices-resource.js

```diff
@@ -128,12 +128,8 @@ export default function InvoicesResource({apiHandler}) {
 
             return apiHandler.getAll(`invoices/${id}/transaction-allocations`, params);
         },
-
-        applyTransaction({id, transactionId, amount = null}) {
-            const data = {
-                transactionId,
-                amount
-            };
+        
+        applyTransaction({ id, data }) {
```

Comment:
This does not follow the existing codestyle `{id, data}`

It is expected?

Reply:
No, it wasn't expected. I fixed the code generator to match the existing codestyle. 👍